src/kernels/jupyter/types.ts

```diff
@@ -360,8 +359,4 @@ export interface IServerConnectionType {
     onDidChange: Event<void>;
 }
 
-export interface IRemoteKernelFinder extends IContributedKernelFinder<RemoteKernelConnectionMetadata> {
-    listKernelsFromConnection(
```

Comment:
This method was only used in tests, hence removed from the interface, not to be used in the extension code.
Was confusing why we had this when we're implemeneting an existing interface